core/src/components/checkbox/checkbox.scss

```diff
@@ -102,6 +102,14 @@
   @include margin(null, null, $checkbox-item-label-margin-bottom, null);
 }
 
+/**
+  * The native control it must be hidden with display:none instead of aria-hidden="true"
+  * to avoid nested interactive elements accessibility issues
+ */
+.checkbox-native-control {
```

Comment:
You should be able to target the `input` here directly without adding a class

Reply:
🫡